sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -204,60 +218,89 @@ func (m *marshaller) addScopeTree(s *ScopeTree) string {
 		EnvironmentId: m.addDefaultEnv(),
 	}
 
-	m.updateIfCombineComposite(s, transform)
+	if err := m.updateIfCombineComposite(s, transform); err != nil {
+		return "", errors.Wrapf(err, "failed to add socpe tree: %v", s)
```

Comment:
Typo I missed on the first pass.
```suggestion
		return "", errors.Wrapf(err, "failed to add scope tree: %v", s)
```